etc/ui-config/mh_default_org/admin-ui/imprint.de.html

```diff
@@ -0,0 +1,9 @@
+<h1>Muster Impressum</h1>
```

Comment:
IMO these should be renamed to something like `sample-imprint.$LANG.html`, or the default contents should be blank when rendered.  We don't have sample data like this showing up anywhere else in the UI, so I can see adopters being confused when this pops up with their next upgrade.

Not having the files needs to be properly handled by the UI anyway, but if you want to keep the files then we should provide blank (with commented out sample maybe) rendering defaults.

Reply:
Hi, I added a configuration to display the footer elements (which defaults to false), hope this resolves your concern.

Reply:
(also see https://github.com/opencast/opencast-admin-interface/pull/306)